settings_base.py

```diff
@@ -68,7 +68,10 @@
                 "wafer.context_processors.menu_info",
                 "wafer.context_processors.registration_settings",
                 "context.context",
-            )
+            ),
+            "builtins": [
+                "django_browser_reload.templatetags.django_browser_reload"
```

Comment:
I'm not sure this is needed. Browser reload started working again as soon as the body tag was closed.

Maybe you want this for something though, spent a bit of time googling, but I'm not sure what it's for.

I suspect it was added while you were trying to fix the issue, but it isn't needed for that.